massgov/openmass
# Dp 24909 style corrections for collections

**Description:**
- Fixes on collection header title.
- Showing banner style only if the description or bg color is not empty.

**Jira:** (Skip unless you are MA staff)
DP-24909

**Testing steps, scenario 1:**
- Edit a top collection term.
- Remove description (if any)
- Remove background color (if any)
- Save
- Visit /collection/[url-name] for that collection term
- Notice there is an "Offered by" and the breadcrumb has a gray bottom border, separating OFFERED BY and the breadcrumb.

**Testing steps, scenario 2:**
- Edit a top collection term.
- Add a description (if any) OR add a background color (if any)
- Save
- Visit /collection/[url-name] for that collection term
- Notice there is NOT an "Offered by" and the breadcrumb has NOT a gray bottom border.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
